tests/suites/test_suite_psa_crypto.data

```diff
@@ -6984,27 +6984,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
 PSA generate key: RSA, 512 bits, good, sign (PKCS#1 v1.5)
```

Comment:
This test case is never executed on the CI because we don't have a build with `MBEDTLS_RSA_MIN_KEY_SIZE <= 512`. Here the key size doesn't matter, we just wanted a small size to not waste CPU time, but a slightly bigger size is no big deal. Please use 1024 bits or `PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS` instead.